src/apps/pfumain.cpp

```diff
@@ -0,0 +1,55 @@
+#include "common/Configuration.hpp"
+#include "handler.hpp"
+#include "pfu.hpp"
+
+#include <fmt/format.h>
+#include <lyra/lyra.hpp>
+#include <spdlog/sinks/stdout_color_sinks.h>
+#include <spdlog/spdlog.h>
+
+extern void boot_db();
+
+// pfu - player file upgrade utility for bulk upgrading all player files
+// as part of a release. Especially where the release contains code changes
+// that will affect character stats or anything else that ought to be
+// persisted in the pfiles.
+// This is intended to be used with the prepare-pfiles.sh wrapper script.
+//
+// There are some limitations right now:
```

Comment:
I've noted some potential things for the future around alternative file formats.